connector-packager/tests/test_jar_signer.py

```diff
@@ -0,0 +1,38 @@
+import unittest
```

Comment:
Is this test_jar_signer.py already in the packaging branch?

Reply:
looks like it isn't did I pull in something new?

Reply:
Ensure you've got 4549b1a

Reply:
My mistake, it's in there. I think it's showing up as "new" in this PR because it's been moved.